CMSgov/easi-app
# EASI-3022/ task list status

# [EASI-3022](https://jiraent.cms.gov/browse/EASI-3022)

## Changes and Description

- Remove separate Admin view from task status
- Rename all task status requester instances to remove requester.

This ticket isn't meant to cover the design of the task status list. This was handled in [EASI-2889](https://github.com/CMSgov/easi-app/pull/2068)


## How to test this change
1. Create a system intake
```gql
mutation createSystemIntake{
  
  createSystemIntake(input:{
      requestType :MAJOR_CHANGES
      requester: {
          name: "Mr. Fun Person"
      }
    } )
  
  {
    id
  }
  
  
}
```
2. Query the intake
```gql
query systemIntake{
  
  systemIntake(id:"{{systemIntakeID}}")
  
  {
      itGovTaskStatuses {      
        intakeFormStatus
        feedbackFromInitialReviewStatus
        decisionAndNextStepsStatus
        bizCaseDraftStatus
        grtMeetingStatus
        bizCaseFinalStatus
        grbMeetingStatus

      }
}
```
3. Ensure you get default statuses (they should like like this
```JSON

{
    "data": {
        "systemIntake": {
            "itGovTaskStatuses": {
                "intakeFormStatus": "READY",
                "feedbackFromInitialReviewStatus": "CANT_START",
                "decisionAndNextStepsStatus": "CANT_START",
                "bizCaseDraftStatus": "CANT_START",
                "grtMeetingStatus": "CANT_START",
                "bizCaseFinalStatus": "CANT_START",
                "grbMeetingStatus": "CANT_START"
            }
        }
    }
}


```


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
